doc/source/file_formats.rst

```diff
@@ -46,9 +46,48 @@ experimental support for NeuroLucida .asc files.
 SWC
 ---
 
-The SWC format represents a neuron as a tree of 3D points with additional information.
+The SWC format represents a neuron as a set of trees that are connected to a soma.
+The neuronal morphology is encoded as a rooted tree of 3D points and the corresponding radii.
 More information can be found `here <http://research.mssm.edu/cnic/swc.html>`_.
 
+The points of type "1" that represent the soma, have to be in the beginning of the file. 
+The soma can be represented in one of the following formats: 
+
+TypeA: One point soma
+^^^^^^^^^^^^^^^^^^^^^^
+
+The soma is represented by its center and a radius that corresponds to the sphere
+that preserves the surface area of the soma. The center of the soma has as parent ID -1 
+and all the initial points of the neuronal trees are connected to the soma. 
+
+TypeB: Three point soma
```

Comment:
No, a bifurcation is still not allowed in any type of some (crazy I know). It depends what you mean by bifurcation. So, we do not accept something like the following:
```
o---o---o---o
    |
    o
    |
    o
```

But the three point some is represented as: 
```
o (Parent_ID = 0)
|
o (Parent_ID = -1)
|
o (Parent_ID = 0)

```
because all the points are connected to the center of the soma.  See the picture from http://neuromorpho.org/SomaFormat.html :

![2](https://cloud.githubusercontent.com/assets/14071443/18380011/ade7a8c0-7675-11e6-908b-c9f1ab817b58.png)